node/request.js

```diff
@@ -198,6 +198,8 @@ TChannelRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
 TChannelRequest.prototype.resend = function resend() {
     var self = this;
 
+    if (self.channel.destroyed) return;
```

Comment:
should that be done firts, then this PR rebased on it and this addition removed?